apps/mull-ui/src/app/components/event-page-info/event-page-info.scss

```
@@ -0,0 +1,30 @@
+@import 'variables';
+
+.info-row {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  padding: 0.5rem 1rem;
+  box-sizing: border-box;
+  align-items: flex-start;
+}
+
+.event-page-info-container {
+  div {
+    border-bottom: 1px solid rgba($color: $primary-color, $alpha: 0.5);
```

Matches better the mockup and the mockups use 0.5 for width.
```suggestion
    border-bottom: 0.5px solid rgba($color: $primary-color, $alpha: 0.5);
```

---

Yup, will also simplify this, since we don't need to exclude the last children from having a border